conf/aws/application.conf

```diff
@@ -0,0 +1,162 @@
+#
+# Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy of the
+# License at  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+
+## Reference configuration properties for the Gobblin AWS cluster launcher
+
+#########################################################################
+## Must change the config properties below, rest can be left as-is     ##
+#########################################################################
+
+# Cluster name
+gobblin.aws.cluster.name=GobblinAWS
+
+# Zookeeper connection string
+# Note: You can use the example below, but the example Zookeeper may or may not be available
+# Example: gobblin.cluster.zk.connection.string="ec2-54-149-223-115.us-west-2.compute.amazonaws.com:2181"
+gobblin.cluster.zk.connection.string="<Zookeeper connection string>"
+
+# AWS Credentials
+gobblin.aws.service.access="<AWS Access Key>"
+gobblin.aws.service.secret="<AWS Secret Key>"
+
+# Leave as-is for sample wikipedia pull job
+# -- OR --
+# Specify S3 URI for job configuration:
+# Steps to obtain this URI:
+# 1. Create a S3 folder in a bucket, say the bucket name is "s3bucket" and folder is "/gobblin-confs/job-conf/"
+# 2. Zip your pull files in a zip and upload to this folder. Say your zip file name is myPullFiles.zip
+# 3. Specify S3 URI for this zip file in the config below
+#    For the description above: "https://s3-region.amazonaws.com/s3bucket/gobblin-confs/job-conf/myPullFiles.zip"
+gobblin.aws.job.conf.s3.uri="https://s3-us-west-2.amazonaws.com/gobblin-libs/job-config/wikipedia.zip"
```

Comment:
Not verbatim but it can be extended, also we should make use of new JobLauncher pushed by Chavdar - as per my understanding it is very much modeled around gobblin-cluster but I need to dig deeper